javascript-timers/timeout.js

```diff
@@ -0,0 +1,5 @@
+function changeHeading(string) {
+  document.querySelector('.message').innerHTML = string;
```

Comment:
`querySelector`s should be done globally rather than inside functions. By including it inside of here, you're forcing us to traverse the DOM every single time this function gets called. By including it globally, we just do the search once, and always have a reference to it.